content/source/docs/cloud/api/variable-sets.html.md

```diff
@@ -0,0 +1,645 @@
+    "type": "vars",
+    "attributes": {
+      "key": "g6e45ae7564a17e81ef62fd1c7fa86138",
+      "value": "61e400d5ccffb3782f215344481e6c82",
+      "description": "new cheeeese",
+      "sensitive": false,
+      "category": "terraform",
+      "hcl": false
+    }
+  }
+}
+```
+
+## Delete a Variable in a Variable Set
+
+`DELETEvarsets/:varset_id/relationships/vars/:var_id`
+
+| Parameter            | Description                      |
+| -------------------- | ---------------------------------|
+| `:varset_id`         | The variable set ID              |
+| `:var_id`            | The ID of the variable to delete |
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request DELETE\
+  --data @payload.json \
+  https://app.terraform.io/api/v2/varsets/varset-4q8f7H0NHG733bBH/relationships/vars/var-EavQ1LztoRTQHSNT
+```
+
+on success, responds with no content
+
+## List Variables in a Variable Set
+
+`GET varsets/:varset_id/relationships/vars`
+
+| Parameter            | Description         |
+| -------------------- | --------------------|
+| `:varset_id`         | The variable set ID |
+
+### Sample Response
+
+```json
+{
+  "data": [
+    {
+      "id": "var-134r1k34nj5kjn",
+      "type": "vars",
+      "attributes": {
+        "key": "F115037558b045dd82da40b089e5db745",
+        "value": "1754288480dfd3060e2c37890422905f",
+        "sensitive": false,
+        "category": "terraform",
+        "hcl": false,
+        "created-at": "2021-10-29T18:54:29.379Z",
+        "description": ""
+      },
+      "relationships": {
+        "varset": {
+          "data": {
+            "id": "varset-992UMULdeDuebi1x",
+            "type": "varsets"
+          },
+          "links": { "related": "/api/v2/varsets/1" }
+        }
+      },
+      "links": { "self": "/api/v2/vars/var-BEPU9NjPVCiCfrXj" }
+    },
+  ],
+  "links": {
+    "self": "app.terrafor.io/app/varsets/varset-992UMULdeDuebi1x/vars",
+    "first": "app.terrafor.io/app/varsets/varset-992UMULdeDuebi1x/vars?page=1",
+    "prev": null,
+    "next": null,
+    "last": "app.terrafor.io/app/varsets/varset-992UMULdeDuebi1x/vars?page=1"
+  }
+}
+```
+
+### Workspaces Relationships
+
+## Apply Variable Set to Workspaces
+
+`POST varsets/:varset_id/relationships/workspaces`
+
+| Parameter            | Description         |
+| -------------------- | --------------------|
+| `:varset_id`         | The variable set ID |
+
+### Sample Payload
+
+```json
+{
+  "data": [
+    {
+      "type": "workspaces",
+      "id": "ws-YwfuBJZkdai4xj9w"
+    }
+  ]
+}
+```
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request POST \
+  https://app.terraform.io/api/v2/varsets/varset-kjkN545LH2Sfercv/relationships/workspaces
+```
+
+On success, this endpoint responds with no content.
+
+## Remove a Variable Set from Workspaces
```

Comment:
This makes it seem like I could do multiple workspaces - but there's just one ID. Should we clarify? CAN you remove from more than one at once, or is it just one at a time?